otter/models/intents.py

```diff
@@ -0,0 +1,16 @@
+from characteristic import attributes
+
+from effect.twisted import deferred_performer
+
+
+@attributes(['scaling_group', 'modifier'])
+class ModifyGroupState(object):
```

Comment:
The `GroupModelRequest` is kind of a bridge between specific intents and `Func`. The only difference is instead of storing the function, you store the function name. But I do like the abstraction you gave of creating specific intents. Out of curiosity, if we were implementing model with effect today, how would you define the intent? This comment is non-blocking though :). I am fine with current intent.